main.py

```diff
@@ -1,12 +1,283 @@
+        name = input("Which customer name would you like to select? ")
+        return customer_list.get_customer(name=name)
+    elif select_by == "id":
+        id = input("Which customer id would you like to select? ")
+        if id.isnumeric():
+            id = int(id)
+            return customer_list.get_customer(id=id)
+    else:
+        print("Could not select. Please enter id or title.")
+
+
+def valid_entry(video_list, select_by):
+    if select_by == "title":
+        title = input("Which video title would you like to select? ")
+        return video_list.get_video(title=title)
+    elif select_by == "id":
+        id = input("Which video id would you like to select? ")
+        if id.isnumeric():
+            id = int(id)
+            return video_list.get_video(id=id)
+    else:
+        print("Could not select. Please enter id or title.")
+
+
+def select_video(video_list):
+    select_by = input("Would you like to select by? Enter title or id: ")
+    result = valid_entry(video_list, select_by)
+    if result:
+        if result == "Could not find customer by that name or id":
+            select_video(video_list)
+        else:
+            video_list.selected_video = result
+            print_stars()
+            print("Selected video: ", video_list.selected_video)
+    else:
+        select_video(video_list)
+
+
+def select_customer(customer_list):
+    select_by = input("Would you like to select by? Enter name or id: ")
+    result = valid_entry_customer(customer_list, select_by)
+    if result:
+        if result == "Could not find customer by that id or title.":
+            select_customer(customer_list)
+        else:
+            customer_list.selected_customer = result
+            print_stars()
+            print("Selected customer: ", customer_list.selected_customer)
+    else:
+        select_customer(customer_list)
+
+
+def list_options():
+
+    options = {
+        "1": "List all videos",
+        "2": "Create a video",
+        "3": "Update selected video",
+        "4": "Delete selected video",
+        "5": "Select a video",
+        "6": "List all customers",
+        "7": "Create a customer",
+        "8": "Update selected customer",
+        "9": "Delete selected customer",
+        "10": "Select a customer",
+        "11": "Check out a video to a customer",
+        "12": "Check in a video from a customer",
+        "13": "List all options",
+        "14": "Quit"
+    }
+
+    print_stars()
+    print("Welcome to the Retro Video Store CLI")
+    print("These are the actions you can perform")
+    print_stars()
+
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    print_stars()
+
+    return options
+
+
+def make_choice(options, video_list, customer_list, rentals_list):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("What would you like to do? Select 14 to see all options again")
```

Comment:
It's 13 to see the options again.

```suggestion
        print("What would you like to do? Select 13 to see all options again")
```